lms/djangoapps/instructor/tests/test_certificates.py

```diff
@@ -1197,26 +1198,6 @@ def test_invalid_user_name_error(self):
         # Assert Error Message
         assert res_json['message'] == f'{invalid_user} does not exist in the LMS. Please check your spelling and retry.'
 
-    def test_user_not_enrolled_error(self):
```

Comment:
Removed this test. With the changes in this PR we will no longer be checking if the learner is actively enrolled in the course-run. I didn't see much use for this check in the certificate invalidation use case.